package.json

```diff
@@ -32,7 +32,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/freenas/webui.git"
+    "url": "https://github.com/truenas/webui.git"
   },
   "bugs": {
     "url": "https://bugs.freenas.org"
```

Comment:
Waiting to hear back from management about this link. There might be something more appropriate for this entry.

Reply:
Done 👍🏻